superset/utils/dashboard_filter_scopes_converter.py

```diff
@@ -77,10 +77,13 @@ def copy_filter_scopes(
 ) -> Dict:
     new_filter_scopes = {}
     for (slice_id, scopes) in old_filter_scopes.items():
-        new_filter_key = old_to_new_slc_id_dict[int(slice_id)]
-        new_filter_scopes[str(new_filter_key)] = scopes
-        for scope in scopes.values():
-            scope["immune"] = [
-                old_to_new_slc_id_dict[slice_id] for slice_id in scope.get("immune")
-            ]
+        new_filter_key = old_to_new_slc_id_dict.get(int(slice_id))
```

Comment:
I realize this logic was in the old code but is `slice_id` a `str` as opposed to an `int`?

Reply:
Yes `filter_scopes` metadata, its key `slice_id` is `str`. `old_to_new_slc_id_dict` is Dict[int, int].

Reply:
@graceguo-supercat if that's the case lines 85 and 87 would need to be updated.